tests/admin_views/tests.py

```diff
@@ -1739,6 +1739,32 @@ def test_history_view(self):
 
             self.client.get(reverse('admin:logout'))
 
+    def test_history_view_verbose_model_name(self):
+        login_url = '%s?next=%s' % (
+            reverse('admin:login'), reverse('admin:index')
+        )
+        change_dict = {
+            'title': 'Ikke fordømt',
+            'content': '<p>edited article</p>',
+            'date_0': '2008-03-18', 'date_1': '10:54:39',
+            'section': self.s1.pk
```

Comment:
including trailing comma in dicts so if more items are added later we don't need to modify this line again